config.py

```diff
@@ -1386,6 +1387,48 @@ def get_t1_from_meeg(bids_path):
     raise ValueError(msg)
 
 
+def check_baseline(baseline: Tuple[Optional[float], Optional[float]],
+                   epochs_tmin: float, epochs_tmax: float) -> None:
```

Comment:
```suggestion
def check_baseline(
    *,
    baseline: Tuple[Optional[float], Optional[float]],
    epochs_tmin: float,
    epochs_tmax: float
) -> None:
```

Reply:
why the *,  in the parameters ?

Reply:
`*` in this case forces all arguments to be keyword arguments (no positional args allowed)